storage/src/backend/kv_store_gced.rs

```diff
@@ -147,14 +163,22 @@ impl<T: 'static + KVStore + Default> KVStore for KVStoreGCed<T> {
         self.current.delete(key)
     }
 
+    /// Marks an entry as "reused" in the current cycle.
+    /// 
+    /// Entries that are reused/referenced in current cycle
+    /// will be preserved after garbage collection.
     fn mark_reused(&mut self, key: EntryHash) {
         let _ = self.msg.lock().unwrap().send(CmdMsg::MarkReused(key));
     }
 
-    fn retain(&mut self, pred: HashSet<EntryHash>) -> Result<(), KVStoreError> {
+    /// Not needed/implemented. Maybe this method should go into separate trait?
```

Comment:
Can you put ah `TODO: ` prefix here? so that it shows up in searches

Reply:
Done.